ntoskrnl/ke/arm/thrdini.c

```diff
@@ -193,8 +193,18 @@ KiIdleLoop(VOID)
             /* The thread is now running */
             NewThread->State = Running;
 
+#ifdef CONFIG_SMP
+            /* Do the swap at SYNCH_LEVEL */
+            KfRaiseIrql(SYNCH_LEVEL);
```

Comment:
Note that the ARM changes currently do "nothing" useful:
https://github.com/reactos/reactos/blob/ab528ac6ae105b19d457587dc680d8016720bc3d/ntoskrnl/include/internal/arm/ke.h#L7